.circleci/config.yml

```diff
@@ -411,6 +411,40 @@ jobs:
           command: ./scripts/ci-job-succeeded.sh
           when: on_success
 
+  publish-storybook:
+    description: "Publish Storybook"
+    executor: grafana-publish
+    steps:
+      - checkout
+      - run:
+          name: CI job started
+          command: ./scripts/ci-job-started.sh
+      - run:
+          name: Publish Storybook
+          command: |
+            echo $GCP_GRAFANA_UPLOAD_KEY > /tmp/gcpkey.json
+            gcloud auth activate-service-account --key-file=/tmp/gcpkey.json
+            yarn install --frozen-lockfile --no-progress
+            yarn storybook:build
+            if [[ $CIRCLE_BRANCH == "chore/test-release-pipeline" ]]; then
+              # We're testing the release pipeline
+              echo Testing release
+            elif [[ $CIRCLE_BRANCH == "master" ]]; then
+              gsutil -m rsync -d -r ./packages/grafana-ui/dist/storybook gs://grafana-storybook/canary
+            elif [[ -n $CIRCLE_TAG ]]; then
+              gsutil -m rsync -d -r ./packages/grafana-ui/dist/storybook gs://grafana-storybook/$CIRCLE_TAG
```

Comment:
Minor: can we also put the release build into `latest` directory?

Reply:
Sounds like a good plan!